ktor-http/ktor-http-cio/common/src/io/ktor/http/cio/HttpHeadersMap.kt

```diff
@@ -23,24 +23,19 @@ private const val EXPECTED_HEADERS_QTY = 64
  * [6] next entry index (multiplied) with the same name hash
  * [7] reserved
  */
-@SharedImmutable
 private const val HEADER_SIZE = 8
-
-@SharedImmutable
 private const val HEADER_ARRAY_POOL_SIZE = 1000
-
-@ThreadLocal
 private val EMPTY_INT_LIST = mutableListOf<Int>()
```

Comment:
some time ago this was migrated from using `IntArray` to lists because of mutations. Looks like it's better to migrate it back to arrays, to not box it all time

Reply:
fixed